app/views/groups/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="transaction-container">
+    <nav class="nav">
+ <%=link_to groups_path do %> <i class="fa-solid fa-chevron-left" style="color: #f3f4f7;"></i>
+<%end%>
+    <h3 class="tans-header"><%="Total: $ #{@total_amount}"%></h3>
+    <i class="fa-solid fa-magnifying-glass" style="color: #fafcff;"></i>
+    </nav>
+    <div class="recent-container">
+        <div class="most-recent">
+            <h4>Most recent</h4>
+            <span></span>
+        </div>
+        <h4 class="ancient">Most ancient</h4>
+    </div>
+<div class="transaction-wrapper">
+<%@group_entities.each do |group_entity|%>
+    <div class= "transaction">
+        
+        <div class= "name" class= "det-name">
+            <h4 class= "amount"><%=group_entity.entity.name%></h4>
+            <h4 class= "date"><%=group_entity.entity.updated_at%></h4>
+            <h4><%="$#{group_entity.entity.amount}"%></h4>
+        </div>
+        
+    </div>
+
+<%end%>
+<div>
+
+<button class="add-catagory"><%=link_to "Add a New Transaction", "/groups/#{params[:id]}/entities/new"%>
+
```

Comment:
- [ ] Please make this button take the whole width of the page and place it at the bottom to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/addisalemseifu/budget-app-on-rails/assets/68796232/e359cd5f-fac9-4fd2-a4b2-263b7e19ae25) | ![image](https://github.com/addisalemseifu/budget-app-on-rails/assets/68796232/539eeb8c-0aa2-45d1-982f-eb0d599a94b9)  |